client/src/components/Workflow/Editor/LintSection.vue

```diff
@@ -1,22 +1,31 @@
 <template>
-    <div class="lint-section">
-        <b-button
-            :aria-expanded="expanded ? 'true' : 'false'"
-            :aria-controls="collapseId"
-            :variant="variant"
-            @click="expanded = !expanded"
-        >
-            <font-awesome-icon icon="check" v-if="okay" />
-            <font-awesome-icon icon="times" v-else />
-            {{ title }}
-            <font-awesome-icon icon="angle-double-up" v-if="expanded" />
-            <font-awesome-icon icon="angle-double-down" v-else />
-        </b-button>
-        <b-collapse :id="collapseId" v-model="expanded">
-            <div class="lint-section-body">
-                <slot> </slot>
+    <div class="mb-2">
+        <div v-if="isOkay">
+            <font-awesome-icon icon="check" class="text-success" />
+            <span>{{ successMessage }}</span>
+        </div>
+        <div v-else>
+            <font-awesome-icon icon="exclamation-triangle" class="text-warning" />
+            <span>{{ warningMessage }}</span>
+            <div v-if="hasWarningItems" class="mt-2">
+                <div
+                    v-for="(item, idx) in warningItems"
+                    :key="idx"
+                    @mouseover="onMouseOver(item.stepId)"
+                    @mouseleave="onMouseLeave(item.stepId)"
```

Comment:
This would be a better abstraction IMO if emitted the item and not the step id - so we could theoretically highlight actual inputs and outputs. That isn't important in this PR, but I wanted to highlight a direction I would want to take this if development continues...

Reply:
Modified it such that item is parsed instead of item.stepId.